CharacterCreation/03_Feats.txt

```diff
@@ -1041,7 +1041,7 @@ attack this turn against a target you are aimed at, you gain +1 Accuracy and +10
 to the weapon's largest range bracket. Miss Chance can go below the -2 minimum
 (see section "Combat:" in the basic rules document). 
```

Comment:
Should we change/remove this since the minimum is no longer listed in basic rules?